helmfile.d/15-ory.yaml

```diff
@@ -0,0 +1,85 @@
+bases:
+- ../environments.yaml
+
+---
+
+releases:
+  - name: kratos
+    chart: ory/kratos
+    version: {{ .Values.kratos._chart_version }}
+    installed: {{ .Values.kratos._install }}
+    timeout: {{ add .Values.base_timeout .Values.kratos._extra_timeout }}
+    values:
+      - "../etc/kratos/values.yaml"
+      - {{ .Values.kratos | toYaml | indent 8 | trim }}
+    set:
+      - name: serverName
+        value: {{ .Values.server_name }}
+      - name: kratos.config.dsn
+        value: postgres://{{ .Values.management_portal.postgres.user }}:{{ .Values.management_portal.postgres.password }}@{{ .Values.management_portal.postgres.host }}:{{ .Values.management_portal.postgres.port }}/kratos
```

Comment:
It's better that the database name is also configurable

Reply:
Done!